pkgs/applications/audio/opustags/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, cmake, pkgconfig, libogg, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "opustags";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "fmang";
+    repo = "opustags";
+    rev = version;
+    sha256 = "09z0cdg20algaj2yyhfz3hxh1biwjjvzx1pc2vdc64n8lkswqsc1";
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_INSTALL_PREFIX=$out"
```

Comment:
Not needed here, cmake hook add it for you:
https://github.com/NixOS/nixpkgs/blob/a6d56b9df9e822413fd4fa9f1e2564b3926d88f9/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L34